### PR TITLE
Updated the Player class (it now passes style cop)

### PR DESCRIPTION
**Pull request by Peter-Scheelke**
Also added unit tests. I need to figure out the merge issue, though.

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/sh4ne/Scrabble_CS3450/pull/43" target="_blank">ReviewNinja

File: EasyNetworkTests/EasyNetworkUnitTests.cs [hunk 1]
```diff
@@ -52,53 +52,53 @@ public class EasyNetworkUnitTests
...
+
+            //Assert.IsTrue(serverObjectsReceived[1].Item1 is OtherTestObject);
+            //OtherTestObject actualObjectTwo = serverObjectsReceived[1].Item1 as OtherTestObject;
+            //Assert.AreEqual(clientTwo.Id, serverObjectsReceived[1].Item2);
+            //Assert.AreEqual(obj2.Value, actualObjectTwo.Value);
+            //Assert.AreEqual(obj2.Text, actualObjectTwo.Text);
+
+            //server.Stop();
+            //clientOne.Stop();
+            //clientTwo.Stop();
```
Comment (sh4ne) [inline EasyNetworkTests/EasyNetworkUnitTests.cs]: why are these tests commented out?

File: ScrabbleUnitTest/ScrabbleUnitTest.csproj [hunk 2]
```diff
@@ -50,10 +50,10 @@
     </Otherwise>
   </Choose>
   <ItemGroup>
-    <Compile Include="FirstThingsFirstUnitTests.cs" />
     <Compile Include="Game Objects\GameBoardSquareTests.cs" />
     <Compile Include="Game Objects\LetterTileRackTests.cs" />
     <Compile Include="Game Objects\LetterTileTests.cs" />
+    <Compile Include="PlayerClass\PlayerTests.cs" />
```
Comment (sh4ne) [inline ScrabbleUnitTest/ScrabbleUnitTest.csproj:57]: Look for your merge conflicts in this file. Open up this file in a text editor. Have you tried merging your branch into the recent sh4ne/master?

File: EasyNetworkTests/EasyNetworkUnitTests.cs (hunk 1 above)
Comment (Peter-Scheelke) [inline EasyNetworkTests/EasyNetworkUnitTests.cs]: They were breaking (not failing).

File: ScrabbleUnitTest/ScrabbleUnitTest.csproj (hunk 2 above)
Comment (Peter-Scheelke) [inline ScrabbleUnitTest/ScrabbleUnitTest.csproj:57]: That's where I figured it was.

**Comment by Peter-Scheelke** [comment]
There we go, the merge issue is gone (what is "FirstThingsFirstUnitTests.cs" anyway?)

**Comment by Peter-Scheelke** [comment]
:+1:

**Comment by sh4ne** [comment]
:+1: the FirstThingsFirstUnitTest.cs was created by John, he erased the reference and should be working in main master.

**Comment by murple13** [comment]
:+1:

**Comment by joshcv2112** [comment]
:+1:

**Pull request merged**